docker-compose.yml

```diff
@@ -139,6 +139,48 @@ services:
     ports:
       - "8082:8082"
     
+  events:
+    build:
+      context: ./services/Events
+      dockerfile: Dockerfile
+    command: uvicorn events_app.main:app --host 0.0.0.0 --port 8006
+    volumes:
+      - ./services/Events:/app
+    ports:
+     - "8006:8006"
+    env_file:
+      - ./services/Events/.env.example
```

Comment:
.env.example is NOT .env

If it doesn't contain real secrets, the Events service won't know:

- DATABASE_URL
- CELERY_BROKER_URL
- RABBITMQ_URL (if needed)
- JWT_SECRET (if the service validates tokens)

 **_In most cases, .env.example should not be used as a prod env._**